RELEASING.md

```diff
@@ -2,69 +2,85 @@
+| KROXYLICIOUS_RELEASE_GPG_PRIVATE_KEY | GPG private key for Kroxylicious (plain text) |
+| KROXYLICIOUS_RELEASE_GPG_PASSPHRASE  | GPG passphrase                                |
+
+
+## Release steps
 
-where `<YOUR_KEY_SHORT_ID>` is the short id of your PGP key
-and `<RELEASE_VERSION>` is a release number such as `0.6.0`.
+Use the [Kroxylicious Team Developers](https://kroxylicious.slack.com/archives/C04V1K6EAKZ) Slack Channel to coordinate
+the release with the other developers.  It is important no other work is merged to main during the release.
 
-The script will push the Maven artefacts to a Nexus **staging repository** and it will open a PR that will commit the release to Git.
+### Stage the Release
 
-The next steps are:
-1.  Validate the staged artefacts.
-2.  If the staging artefacts are **good**, release the stage repository.  If they are not, **drop** the stage repository and start the release over.
-3.  Merge the PR.
+Run [stage_workflow](https://github.com/kroxylicious/kroxylicious-junit5-extension/actions/workflows/stage_release.yml).
+Set the `release-version` argument to the version being release e.g. `0.7.0`.
 
-These are described below.
+This will:
 
-### Validating the staged artefacts
+* raise single PR that will contain two commits:
+  1. the first will version the artefacts at `release-version`.  A `release-version` tag will point at this commit.
+  2. the second will re-open main for development, at the next snapshot.
+* stage a release [Nexus UI](https://s01.oss.sonatype.org/). It'll be named `iokroxylioustesting-nn`.
+
+### Verify the Release
 
 You can validate the staged artefacts by having a test application (we'll call it `T`) that uses kroxylicious-junit5-extension use the Maven artefacts by making
 temporary (local) changes to its POM.
 
 1. Find the staging repository in the [Nexus UI](https://s01.oss.sonatype.org/). It'll be named `iokroxylioustesting-nn`.
-2. [Close](https://help.sonatype.com/repomanager2/staging-releases/managing-staging-repositories) the staging respository.  This gives the staging repository a publicly accessible URL.
-3. Add a [`<repositories>`](https://maven.apache.org/pom.html#Repositories) that references the staging repository public url to `T`'s POM.
-4. Update `T`'s kroxylicious-junit5-extension dependency to refer to the `<RELEASE_VERSION>`.
-5. Run `T` build/test cycle but use an alternative cache location to be sure artefacts are being fetched.  Check the build output, you'll see the
-   kroxylicious-junit5-extension come from the staging location. 
+2. Add a [`<repositories>`](https://maven.apache.org/pom.html#Repositories) that references the staging repository public url to `T`'s POM.
+3. Update `T`'s kroxylicious-junit5-extension dependency to refer to the `<RELEASE_VERSION>`.
+4. Run `T` build/test cycle but use an alternative cache location to be sure artefacts are being fetched.  Check the build output, you'll see the
```

Comment:
Probably worth adding a sample command line for that.

Reply:
I did add a command line, is this not enough?

```
MAVEN_OPTS="-Dmaven.repo.local=/tmp/repository" mvn verify
```

Reply:
No that's perfect, it was me missing it in the diff